cmd/ops_agent_windows/run_windows.go

```diff
@@ -153,6 +180,7 @@ func (s *service) generateConfigs() error {
 			return err
 		}
 	}
+
```

Comment:
Why the extra new line?

Reply:
During development probably I had a function call there before. Removed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1051/commits/fdd3b580a5732768b919f9ebc58d02099bcfa8fd .